test/powershell/engine/ExperimentalFeature/EnableDisable-ExperimentalFeature.Tests.ps1

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Enable-ExperimentalFeature and Disable-ExperimentalFeature tests" -tags "Feature","RequireAdminOnWindows" {
+
+    BeforeAll {
+        $systemConfigPath = "$PSHOME/powershell.config.json"
+        if ($IsWindows) {
+            $userConfigPath = "~/Documents/powershell/powershell.config.json"
+        }
+        else {
+            $userConfigPath = "~/.config/powershell/powershell.config.json"
+        }
+
+        $systemConfigExists = $false
+        if (Test-Path $systemConfigPath) {
+            $systemConfigExists = $true
+            Move-Item $systemConfigPath "$systemConfigPath.backup" -Force -ErrorAction SilentlyContinue
+        }
+
+        $userConfigExists = $false
+        if (Test-Path $userConfigPath) {
+            $userConfigExists = $true
+            Move-Item $userConfigPath "$userConfigPath.backup" -Force -ErrorAction SilentlyContinue
+        }
+
+        $testModulePath = Join-Path -Path $PSScriptRoot -ChildPath "assets"
+        $originalModulePath = $env:PSModulePath
+        $env:PSModulePath = $testModulePath
+    }
+
+    AfterAll {
+        if ($systemConfigExists) {
+            Move-Item "$systemConfigPath.backup" $systemConfigPath -Force -ErrorAction SilentlyContinue
+        }
+
+        if ($userConfigExists) {
+            Move-Item "$userConfigPath.backup" $userConfigPath -Force -ErrorAction SilentlyContinue
+        }
+
+        $env:PSModulePath = $originalModulePath
+    }
+
+    AfterEach {
+        Remove-Item $systemConfigPath -Force -ErrorAction SilentlyContinue
+        Remove-Item $userConfigPath -Force -ErrorAction SilentlyContinue
+    }
+
+    It "Enable-ExperimentalFeature will enable Experimental Feature for scope: <scope>" -TestCases @(
+        @{ scope = "AllUsers" },
+        @{ scope = "CurrentUser" }
+    ) {
+        param ($scope)
+
+        $feature = pwsh -noprofile -output xml -command Get-ExperimentalFeature ExpTest.FeatureOne
```

Comment:
This may be a silly idea, but if the experimental feature had an implementation, we could test it here and make sure that all the enabling logic and all the discovering logic always agree. (I know that theoretically `$feature.Enabled` does that, but to guard against regressions we should make sure that `$feature.Enabled` is accurate)

Reply:
The ExperimentalFeature.Basic.Tests.ps1 validates that features are actually enabled/disabled correctly.  These tests is really about the config.